### ISSUE TITLE
report error if build kn-plugin-admin without kubeconfig specified 

### ISSUE THREAD

**Issue by zhanggbj** [knative-sandbox/kn-plugin-admin#19]
If no kubeconfig specified, there's an error msg while go build, looks like it is for operater mod, please help to take a look, thanks!

```
[zhanggong@zhanggongdeMacBook-Pro kn-plugin-admin (as_pr_final ✗)]$ ./hack/build.sh   
🚒 Update
✋ No 'goimports' found. Please use
✋   go install golang.org/x/tools/cmd/goimports
✋ to enable import cleanup. Import cleanup skipped.
🧹 Format
⚖️ License
🚧 Compile
🧪 Test
────────────────────────────────────────────
failed to get installation method: Get "https://192.168.99.102:8443/api/v1/namespaces/knative-serving/configmaps/config-domain": dial tcp 192.168.99.102:8443: i/o timeout

```

**Comment by lanceliuu** [comment]
I cannot reproduce the error. Could you help to provide more details? I have no KUBECONFIG or other context defined while testing.
```./hack/build.sh
🚒 Update
✋ No 'goimports' found. Please use
✋   go install golang.org/x/tools/cmd/goimports
✋ to enable import cleanup. Import cleanup skipped.
🧹  Format
⚖️  License
🚧 Compile
🧪 Test
────────────────────────────────────────────
Version:      v20201122-local-e1f827b-dirty
Build Date:   2020-11-22 14:22:08
Git Revision: e1f827b
```

**Comment by chaozbj** [comment]
No sure if it has the same root cause with the #32, I actually addressed #32 in our old repository, maybe we lost these changes when we moved it to the new repository. I will keep eye on it when I fix the #32 issue.

**Comment by zhanggbj** [comment]
/assign @chaozbj 
Merged!
